lib/linux.js

```diff
@@ -41,18 +41,23 @@ var LinuxBuilder = {
       linux.archName = 'x86_64';
     }
 
-    var tmpFolder = path.join( os.tmpDir(), linux.executable );
+    var outFilename = linux.executable + '-' +
```

Comment:
@stefanjudis Maybe we can set maximum line length to 160? It is a bit of annoying to use 80.